### DISCUSSION TITLE
People stealing code

### DISCUSSION THREAD

**Discussion by rawandahmad698** [acheong08/ChatGPT#159] [General]
@acheong08 Couldn't find a suitable way to contact you..

Take a look at this guy 
https://github.com/labteral/chatgpt-python/issues/7

Literally stole all the methods/code from my lib, renamed the functions and licenses it under a GPL-3.0

**Comment by acheong08** [comment]
@rawandahmad698 A copyright strike is probably necessary.

**Reply by acheong08** [reply]
I also got a few of these clones of my repository but I'm not really concerned.  (Reason being I didn't bring anything significantly new. The chat protocol was relatively simple)

**Reply by rawandahmad698** [reply]
You don't know how much of my time was needed to do the automatic login on Auth0. It was a lot. I won't sit and relax with seeing someone steal it, rename it and license it in a different way without crediting the original work. Very unprofessional

**Reply by acheong08** [reply]
Yeah. Your work is amazing. A copyright strike is 100% necessary. 

Is there a way to report this via GitHub?

**Reply by rawandahmad698** [reply]
Thank you sir, I'm filling out the form now

**Reply by acheong08** [reply]
What's worse is that they financially benefit from this via "buy me a coffee"

**Reply by rawandahmad698** [reply]
Thanks, Takedown report sent

**Reply by acheong08** [reply]
Regarding licensing, how do you deal with it? I have a lot of contributors and I'm afraid that I might accidentally violate some license. I am inexperienced with managing large projects so need some advice.

**Reply by rawandahmad698** [reply]
We can have a discussion via anything but email. Do you have Discord?

**Reply by acheong08** [reply]
I do though I rarely use it. It's Churchless#2499

**Reply by rawandahmad698** [reply]
Added you.
